packages/gnome-themes-extra/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://gitlab.gnome.org/GNOME/gnome-themes-extra
+TERMUX_PKG_DESCRIPTION="This is the version of Adwaita, the standard GNOME theme, for the GTK+ 2/3 toolkit."
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="Yisus7u7 <dev.yisus@hotmail.com>"
+TERMUX_PKG_VERSION=3.28
+TERMUX_PKG_SRCURL="http://deb.debian.org/debian/pool/main/g/gnome-themes-extra/gnome-themes-extra_${TERMUX_PKG_VERSION}.orig.tar.xz"
+TERMUX_PKG_SHA256=7c4ba0bff001f06d8983cfc105adaac42df1d1267a2591798a780bac557a5819
+TERMUX_PKG_DEPENDS="gtk2, gtk3, glib, libcairo, gtk2-engines-murrine"
```

Comment:
According to the description in the `$TERMUX_PKG_HOMEPAGE`, this is for GTK+ 2, not depending on `gtk3`.

Reply:
> According to the description in the `$TERMUX_PKG_HOMEPAGE`, this is for GTK+ 2, not depending on `gtk3`.

![Screenshot_20220329-210525](https://user-images.githubusercontent.com/64093255/160736149-5526c1e1-ab2c-4a9c-97ce-313ac387ff04.png)


Compiled with gtk3 support.

Reply:
You should check the content of those files. It can be removed I suppose.

Reply:
> You should check the content of those files. It can be removed I suppose.

No, you can check the file 'configure', in the options, if gtk3 is installed gtk3 support will be enabled. 

You can check the package of should, it also comes with gtk3. 

I remind you that termux does NOT provide the adwaita theme, so this is not a duplication.

Reply:
In fact, Adwaita engine for GTK+ 3 is not installed by this package; I see only `lib/gtk-2.0/2.10.0/engines/libadwaita.so`.

I suppose `--enable-gtk3-engine` configure option is for backward compatibility. I would even consider specifying `--disable-gtk3-engine`.

Reply:
> In fact, Adwaita engine for GTK+ 3 is not installed by this package; I see only `lib/gtk-2.0/2.10.0/engines/libadwaita.so`.
> 
> I suppose `--enable-gtk3-engine` configure option is for backward compatibility. I would even consider specifying `--disable-gtk3-engine`.

Sorry, you don't get the point, gtk3 doesn't need the engine to use adwaita. 

The package provides the Adwaita and Adwaita-Dark theme to gtk3 as well, if you want, do your research, and you will realize that there is no other way to get the Adwaita theme in gtk3. 

You can also review the gnome-themes-extra package in the Alpine Linux distribution, the package only provides the gtk3 theme files. 

The functionality of the package is simply being extended, and from what I understand that is not prohibited in termux. 

Apart from that as I said before, there is no other way to get the Adwaita theme at the moment.